gradle/libs.versions.toml

```diff
@@ -1,32 +1,33 @@
 [versions]
 # Kotlin
-kotlin = "1.9.10"
+kotlin = "1.9.22"
```

Comment:
🤔 We were on 1.2.2 of harmony before, so maybe my earlier comment about Kotlin update is less concerning.  I know we had a heck of a time with getting to Kotlin 1.8.x and AGP 8

Reply:
Upgrading these plugins aren't an issue for consumers of Harmony, unless you are compiling from source.